examples/run_all_examples.py

```diff
@@ -0,0 +1,25 @@
+"""Call all *.py files in the working directory.
+"""
+import glob
+import subprocess
+import sys
+
+if __name__ == "__main__":
+    files_not_completed = []
+
+    for file in glob.glob("*.py"):
+        if file == "call_all_examples.py":
```

Comment:
```suggestion
        if file == "run_all_examples.py":
```
maybe you meant to do this?

Reply:
Yes , the filename was originally different and after the renaming i have forgotten to adapt this line. Nice catch